### ISSUE TITLE
JAWS does not use the lang attribute when announcing the title attribute of an element

### ISSUE THREAD

**Issue by OwenEdwards** [FreedomScientific/standards-support#129]
### Summary

JAWS does not use the lang attribute of an element to select the speech synthesizer for the title attribute (or the Acc Description).

````html
<a href="#" title="Changer de langue en français" lang="fr-CA">
    <span>Français</span>
</a>
````

JAWS uses the contents of the `<span>` as the Acc Name, and the `title` attribute as the Acc Description.

### Expected result

When you Tab onto the link, JAWS pronounces both the Acc Name and Acc Description in a French accent, with the role of "link" in an English accent (or whatever the JAWS language is set to).

### Actual result

When you Tab onto the link, JAWS pronounces the Acc Name in a French accent, then the role ("link"), then the Acc Description in an English accent (which is pretty horrible for this text in French!)

### Example

See https://output.jsbin.com/yezaqic

### Additional Information

See https://www.w3.org/International/questions/qa-html-language-declarations#contentvsattribute, where the "bad" example is essentially exactly the same as this example.

#### JAWS version and build number

Version 2018.1808.10
 
#### Operating System and version

Windows 7

#### Browser and version:

IE 11 or Firefox 62.0.3

**Comment by bart-simons** [comment]
similar to issue 8 where @hanshillen commented that he filed internally a bug 97133. No update on this so far. We can only confirm that this has not been addressed until JAWS version 2019.1907.31.

**Comment by stevefaulkner** [comment]
tested with JAWS 2023.2307.37 WORKS AS expected, both acc name and description announced with French accent.
